service/src/test/java/uk/gov/hmcts/reform/fpl/validation/validators/HasGenderValidatorTest.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.hmcts.reform.fpl.validation.validators;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.validation.beanvalidation.LocalValidatorFactoryBean;
+import uk.gov.hmcts.reform.fpl.enums.ChildGender;
+import uk.gov.hmcts.reform.fpl.model.ChildParty;
+
+import java.util.List;
+import javax.validation.ConstraintViolation;
+import javax.validation.Validator;
+
+import static java.util.stream.Collectors.toList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = LocalValidatorFactoryBean.class)
```

Comment:
just curious why we preferred `LocalValidatorFactoryBean` and not `ValidationAutoConfiguration`?